yellowbrick/features/histogram.py

```diff
@@ -0,0 +1,202 @@
+
+# yellowbrick.features.histogram
+# Implementations of histogram with vertical lines to help with balanced binning.
+#
+# Author:   Juan L. Kehoe (juanluo2008@gmail.com)
+# Author:   Prema Damodaran Roman (pdamo24@gmail.com)
+
+# Created:  Tue Mar 13 19:50:54 2018 -0400
+#
+# Copyright (C) 2018 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: histogram.py
+
+"""
+Implements histogram with vertical lines to help with balanced binning.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+import matplotlib.pyplot as plt
+import numpy as np
+
+from yellowbrick.features.base import FeatureVisualizer
+from yellowbrick.exceptions import YellowbrickValueError
+from yellowbrick.utils import is_dataframe
+
+##########################################################################
+## Balanced Binning Reference
+##########################################################################
+
+class BalancedBinningReference(FeatureVisualizer):
+    """
+    BalancedBinningReference allows to generate a histogram with vertical lines
+    showing the recommended value point to bin your data so they can be evenly
+    distributed in each bin.
+
+    Parameters
+    ----------
+    ax : matplotlib Axes, default: None
+        This is inherited from FeatureVisualizer and is defined within
+        BalancedBinningReference.
+    feature : string, default: None
```

Comment:
With the change to the target module, I would recommend that the `feature` attribute is no longer needed, since the target will be passed to the histogram directly!